src/components/resumes/ResumeList/ResumeListAPI.js

```diff
@@ -0,0 +1,78 @@
+import axios from "axios";
+
+const instance = axios.create({
+  baseURL: "https://jsonplaceholder.typicode.com",
+});
+// when the connection to the backend API is made, the corresponding Json will be placed.
+export default async function getResumeList() {
+  let errorRevise = true;
+  try {
+    let result = await instance.get("/users").data;
```

Comment:
Remove this line.
You're mutating the result array either way in the code below.

Reply:
remove this please